lib/blender/drivers/common.rb

```diff
@@ -0,0 +1,26 @@
+module Common
```

Comment:
can you change the module name to SSHExec , and keep it under Blender::Driver namespace ?